src/violations.rs

```diff
@@ -5970,6 +5970,19 @@ impl AlwaysAutofixableViolation for DupeClassFieldDefinitions {
     }
 }
 
+define_violation!(
+    pub struct PreferUniqueEnums;
+);
+impl Violation for PreferUniqueEnums {
+    fn message(&self) -> String {
+        "Consider using removing dupe values.".to_string()
```

Comment:
I know this is copied over from `flake8-pie` but I think we can do something more descriptive...

Can we change `pub struct PreferUniqueEnums` to:

```rs
pub struct PreferUniqueEnums {
  pub value: String
}
```

Then, change this to, like: `format!("Enum contains duplicate value: `{value}`")`.

In `prefer_unique_enums`, you can pass `violations::PreferUniqueEnums(unparse_expr(&values[i], checker.stylist))` to get the formatted value.

Does that seem ok? Let me know if you have any questions!

Reply:
it seems more explicit, which is a good thing.  I've uploaded the review changes 👍🏿